README.md

```diff
@@ -0,0 +1,70 @@
+# OFMT is a formatting and linting tool
+
+* **Formatting** is done solely by prettier. It is completely automated, and not configurable in any way other than changing the [`.prettierrc`](https://github.com/quesabe/ofmt/blob/add-readme/package.json) of this repo, and republishing the package.
+* **Linting** is not related to formatting in any way. The goal of linting is to catch early code quality issues, and enforce usage of best practices.
+
+## Install
+
+```shell
+npm install @ottofeller/ofmt --save-dev
+```
+
+## Configs
+
+The project contains the following configs:
+- `.prettierrc` formatting rules
+- `eslint.quality.cjs` with JS/TS-specific code quality and best practices rules
+- `eslint.tailwind.cjs` same as above, CSS-specific
+
+Direct use of configs is meant only for IDE support. For CLI runs use the provided executables.
+
+## Executables
+
+There are two _bin_ files that perform the code checking:
+- `ofmt` - prettier formatter
+- `olint` - code quality and best practices linter.
+
+### ofmt
+
+#### Options:
+- `-l, --lint` - a flag to perform checking only. Without the flag prettier will rewrite files with fixed formatting.
+- _non-flag_ - the prettier target files. This can contain any of file paths, directory paths, and glob patterns.
+- `install` - a non-flag argument that creates a symlink to prettier config file and adds/extends eslint config within _package.json_. The second argument is the destination path.\
+NOTE: the `install` script does not attempt to overwrite existing `prettier` config - if a _.prettierrc_ file exists it is left as is. An existing `eslint` config is extended with _eslint.quality.cjs_.
+
+#### Examples:
+
+Check TS files with prettier
+```bash
+npx ofmt -l './src/**/*.ts'
```

Comment:
@quesabe it should be `@ottofeller/ofmt`:
<img width="679" alt="image" src="https://user-images.githubusercontent.com/58164/164744196-eb2700c6-a6d1-4d1d-9ef7-5d73a43e97a6.png">

Reply:
Basically it depends on environment:
- if `ofmt` is installed somewhere on `nom` search path it is enough to call a bin;
- if `ofmt` is not installed a `--package` option is required with the name `@ottofeller/ofmt`;
- the call to `@ottofeller/ofmt` also works because there is a bin with the same name as the project, however this way one can't call the `olint` file.

Reply:
@quesabe then we probably need to merge ofmt and olint into a single bin script, so that we have just ofmt and call  it simply with `@ottofeller/ofmt`, which formats the file as a default command, and lints (prettier and code quality) with `lint` command.

If you don't find any issues with this approach, then can you please create a task for the next sprint for this merge? I'm going to merge the docs in their current state, but we can amend them after the merge of ofmt and olint.